k8s/configmap.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: django-config
+  namespace: forum
+data:
+  # Django
+  DJANGO_SETTINGS_MODULE: "core.settings.production"
+  DEBUG: "False"
+  DOCS_ENABLED: "True"
+  ALLOWED_HOSTS: "127.0.0.1,localhost,forum.local"
```

Comment:
ALLOWED_HOSTS is a comma-separated string, not a YAML list. Convert to a YAML list for Django compatibility.

```
ALLOWED_HOSTS:
  - "127.0.0.1"
  - "localhost"
  - "forum.local"
```